src/main/java/org/assertj/core/api/AbstractByteAssert.java

```diff
@@ -375,7 +376,43 @@ public S isCloseTo(Byte expected, Offset<Byte> offset) {
     bytes.assertIsCloseTo(info, actual, expected, offset);
     return myself;
   }
-  
+
+  /** {@inheritDoc} */
```

Comment:
Unfortunately, you can't inherit javadoc as it shows a code example with `double`.
this comment is relevant in other Assert classes